lib/core/sync_job_runner.rb

```diff
@@ -24,7 +24,7 @@ class SyncJobRunner
 
     def initialize(connector_settings)
       @connector_settings = connector_settings
-      @ingester = Core::Ingestion::Ingester.new(Core::Ingestion::EsSink.new(connector_settings.index_name, @connector_settings.request_pipeline))
+      @ingester = Core::Ingestion::EsSink.new(connector_settings.index_name, @connector_settings.request_pipeline)
```

Comment:
```suggestion
      @sink = Core::Ingestion::EsSink.new(connector_settings.index_name, @connector_settings.request_pipeline)
```

Nitpick: any reason to keep the `ingester` terminology?

Reply:
That is a bad merge, thanks for noticing!